package.json

```diff
@@ -139,8 +138,8 @@
     "pretty-bytes": "4.0.2",
     "prop-types": "15.6.0",
     "react-addons-shallow-compare": "15.6.2",
+    "react-dates": "15.5.3",
```

Comment:
@cvializ now that `react-dates` has moved to `devDependencies`, it should probably be removed from `greenkeeper -> ignore` (line 38) so that it can be periodically updated like the other packages.

Reply:
For now while it is experimental I would prefer to manually increment the version number, but I will keep in mind to remove it from greenkeeper ignore before it is launched.